public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,188 @@
+
+```
+$ heroku buildpacks:set https://github.com/MartinNowak/heroku-buildpack-d
+```
+By default the buildpack uses the latest dmd compiler. 
+It is possible to use gdc or ldc and to choose a specific compiler versions by adding a .d-compiler file to your project. 
+Use dmd, ldc, or gdc to select the latest or dmd-2.068.2, ldc-0.16.0, or gdc-4.9.2 to 
+select a specific version of a compiler.
+
+### Deploy the code 
+
+Proceed in your usual git habit and write awesome code. 
+
+To release a new version, just push the newest version to the heroku endpoint.
+
+```
+$ git add .
+$ git commit -am "My first vibe.d release"
+$ git push heroku master
+Counting objects: 9, done.
+Delta compression using up to 2 threads.
+Compressing objects: 100% (6/6), done.
+Writing objects: 100% (9/9), 997 bytes, done.
+Total 9 (delta 0), reused 0 (delta 0)
+
+-----> Fetching custom git buildpack... done
+-----> D (dub package manager) app detected
+-----> Building libevent
+-----> Building libev
+-----> Downloading DMD
+-----> Downloading dub package manager
+-----> Setting PATH:
+-----> Initializing toolchain
+-----> Building app
+       Running dub build ...
+Building configuration "application", build type release
+Running dmd (compile)...
+Compiling diet template 'index.dt' (compat)...
+Linking...
+       Build was successful
+-----> Discovering process types
+       Procfile declares types -> web
+-----> Compiled slug size: 3.5MB
+-----> Launching... done, v4
+       https://rocky-hamlet-67506.herokuapp.com/ deployed to Heroku
+To git@heroku.com:rocky-hamlet-67506.git
+ * [new branch]      master -> master
+```
+
+Open the app in the browser with the following
+
+```
+$ heroku open
+```
+
+### Openning dynos to request 
+
+After deploying, the app is running on a web dyno. 
+Think of a dyno as a lightweight container that runs the command specified in the Procfile.
+
+To check how many dynos are running using the ps command:
+
+```
+$ heroku ps
+Free dyno hours quota remaining this month: 550h 0m (100%)
+For more information on dyno sleeping and how to upgrade, see:
+https://devcenter.heroku.com/articles/dyno-sleeping
+
+No dynos on ⬢ rocky-hamlet-67506
+```
+
+By default, the app is deployed on a free dyno which doesn't access request. 
+Free dynos will sleep after a half hour of inactivity. This causes a delay of a few seconds for the first request upon waking. 
+
+To start the dyno run the following
+
+```
+$ heroku ps:scale web=1
+```
+ 
+### See the logs 
+
+Heroku treats logs as streams of time-ordered events aggregated from the output streams of all your app and Heroku components, 
+providing a single channel for all of the events.
+
+```
+$ heroku logs --tail
+```
+
+
+## More informations 
+
+After deploying your the to Heroku you can make it more awesome by using add-ons.
+
+- [Postgresql](https://elements.heroku.com/addons/heroku-postgresql)
+- [MongoDb](https://elements.heroku.com/addons/mongohq)
```

Comment:
I guess leaving the two is better. 
You can see right away that Heroku have differents type of db that you can use with your vibed app without caring too much.